frontend/src/components/pages/RegistrantExperience/PersonalInfo/CamperCard.tsx

```diff
@@ -0,0 +1,327 @@
+    }
+
+    return () => {
+      // Passing the same reference
+      if (nextBtnRef && nextBtnRef.current) {
+        nextBtnRef.current.removeEventListener("click", updateFormErrorMsgs);
+      }
+    };
+  }, [camper]);
+
+  function AlertDialogExample() {
+    const toast = useToast();
+    const { isOpen, onOpen, onClose } = useDisclosure();
+    const cancelRef = React.useRef(null);
+
+    return (
+      <>
+        <Button
+          color="text.critical.100"
+          variant="outline"
+          onClick={onOpen}
+          colorScheme="blue"
+        >
+          <DeleteIcon />
+          <Text textStyle={{ sm: "xSmallBold", lg: "buttonSemiBold" }} pl={3}>
+            Remove
+          </Text>
+        </Button>
+
+        <AlertDialog
+          isCentered
+          isOpen={isOpen}
+          leastDestructiveRef={cancelRef}
+          onClose={onClose}
+        >
+          <AlertDialogOverlay backgroundColor="overlay.dark.50">
+            <AlertDialogContent>
+              <AlertDialogHeader fontStyle="heading">
+                Remove Camper
+              </AlertDialogHeader>
+
+              <AlertDialogBody>
+                <Text textStyle="bodyRegular">
+                  Are you sure you want to remove this camper from the
+                  registration form? <br />
+                  <br />
+                  <Text as="span" textStyle="bodyBold">
+                    Note: this action is irreversible.{" "}
+                  </Text>
+                </Text>
+              </AlertDialogBody>
+
+              <AlertDialogFooter>
+                <Button ref={cancelRef} onClick={onClose}>
+                  Cancel
+                </Button>
+                <Button
+                  colorScheme="red"
+                  onClick={() => {
+                    setPersonalInfo({
+                      type: PersonalInfoActions.DELETE_CAMPER,
+                      camperId,
+                    });
+                    toast({
+                      description: `${camper.firstName} ${camper.lastName} has been successfully removed`,
+                      status: "success",
+                      duration: 6000,
+                      isClosable: true,
+                      variant: "subtle",
+                    });
+                    onClose();
+                  }}
+                  ml={3}
+                >
+                  Delete
+                </Button>
+              </AlertDialogFooter>
+            </AlertDialogContent>
+          </AlertDialogOverlay>
+        </AlertDialog>
+      </>
+    );
+  }
+
+  return (
+    <Box boxShadow="lg" rounded="xl" borderWidth={1} width="100%">
+      <Box backgroundColor="#FFFFFF" rounded="xl">
+        <Heading textStyle="displayLarge">
+          <Flex py={6} px={{ sm: "5", lg: "20" }} alignItems="center">
+            <Text textStyle={{ sm: "xSmallBold", lg: "displayLarge" }}>
+              Camper #{camperId + 1}
```

Comment:
can we rename this `camperId` to `camperIndex`? Since this is technically not a db id, calling it id is a little confusing

Reply:
Yep!